hack/deploy-wcp.sh

```diff
@@ -176,8 +176,9 @@ function sv_get_cp_ips() {
 function sv_copy_and_load_image() {
     local svip=$1 image=$2
 
+    remote_image_path="/root/$(basename $image)"
     sv_cp_scp_cmd "$svip" "$image"
-    sv_cp_ssh_cmd "$svip" "ctr -n=k8s.io images import '$image'"
```

Comment:
Something like `${image##*/}` instead?

This standalone change is largely moot - the `image-build-amd64` target doesn't use the tag that this script assumes: see the comment around IMAGE_REF. I usually just podman save with the expected ref but cool if we want to start to extract the ref from the image.

Reply:
If you think if it's too much I can try just pick up the latest image from `podman image list` output